app/components/data/shared/job-id/job-id.component.spec.ts

```diff
@@ -1,81 +1,86 @@
-import { Component, DebugElement, NO_ERRORS_SCHEMA } from "@angular/core";
```

Comment:
commented out this entire fixture due to it breaking the build with the recent addition of the "timer(delay)". need Tim's assistance to fix it.